### PR TITLE
6play goes international

### PR DESCRIPTION
**Pull request by omansour**

### CODE REVIEW

File: _posts/2018-03-26-6play-goes-international.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+![RTL Play Belgium](/images/posts/6play-intl/rtl_play_be.png)
+
+## RTL Play (Croatia)
+
+![RTL Play Croatia](/images/posts/6play-intl/rtl_play_hr.png)
+
+## RTL Most (Hungary)
+
+![RTL Play Croatia](/images/posts/6play-intl/rtl_play_hr.png)
```
Comment (DocRoms) [inline _posts/2018-03-26-6play-goes-international.md]: RTL most `Hungary`  and `rtl_hu` ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+tags: [6play]
+image:
+  feature: 6play-intl/feature.png
+  credit: m6web
+comments: true
+language: en
+---
+
+
+Within less than a years m6web and [techm6web](https://twitter.com/TechM6Web) manage to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe based on the [6play](https://www.6play.fr) technology.
```
Comment (pmartin) [inline _posts/2018-03-26-6play-goes-international.md]: "a years" => "a year"
ajouter une virgule après "a year"
"manage" => "managed"
"in Europe based" => "in Europe, all based" ?

Comment (fragab) [inline _posts/2018-03-26-6play-goes-international.md]: a year ?
managed ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  feature: 6play-intl/feature.png
+  credit: m6web
+comments: true
+language: en
+---
+
+
+Within less than a years m6web and [techm6web](https://twitter.com/TechM6Web) manage to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include :
```
Comment (pmartin) [inline _posts/2018-03-26-6play-goes-international.md]: pas d'espace avant les symboles de ponctuation hauts, en anglais

File: _posts/2018-03-26-6play-goes-international.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+comments: true
+language: en
+---
+
+
+Within less than a years m6web and [techm6web](https://twitter.com/TechM6Web) manage to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include : 
+
+* broadcasted and thematics channels,
```
Comment (fragab) [inline _posts/2018-03-26-6play-goes-international.md]: themathic ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* live streaming and video catchup encoding and playout,
+* video resuming,
+* local adservers (for videos and display), DMP system and CDN,
+* contribution via our backoffice and automated thought our API,
```
Comment (pmartin) [inline _posts/2018-03-26-6play-goes-international.md]: "thought" => "through"

File: _posts/2018-03-26-6play-goes-international.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "6play goes international"
+description: ""
+author:
+  name:  Olivier Mansour
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: TechM6Web du coup ici vu le coté "global de l'annonce" ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "6play goes international"
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: `the 6play platform goes international`?

File: _posts/2018-03-26-6play-goes-international.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+description: ""
+author:
+  name:  Olivier Mansour
+  avatar: oliviermansour.jpg
+  email:
+  twitter:  omansour
+  facebook:
+  github: omansour
+category: 6play
+tags: [6play]
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: +tags `intl`si on fait d'autres articles (ce que j'espère) sur le sujet

File: _posts/2018-03-26-6play-goes-international.md (hunk 3 above)
Comment (Oliboy50) [inline _posts/2018-03-26-6play-goes-international.md]: `These` (c'est déjà le pluriel de `this` je crois)

File: _posts/2018-03-26-6play-goes-international.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  feature: 6play-intl/feature.png
+  credit: m6web
+comments: true
+language: en
+---
+
+
+Within less than a year m6web and [techm6web](https://twitter.com/TechM6Web) managed to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe, all based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include:
```
Comment (Oliboy50) [inline _posts/2018-03-26-6play-goes-international.md]: `These` (c'est déjà le pluriel de `this` je crois)

File: _posts/2018-03-26-6play-goes-international.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+language: en
+---
+
+
+Within less than a years m6web and [techm6web](https://twitter.com/TechM6Web) manage to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include : 
+
+* broadcasted and thematics channels,
+* live streaming and video catchup encoding and playout,
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: s/ live streaming and video catchup encoding and playout/ live streaming, video catchup encoding and playout

File: _posts/2018-03-26-6play-goes-international.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+  credit: m6web
+comments: true
+language: en
+---
+
+
+Within less than a years m6web and [techm6web](https://twitter.com/TechM6Web) manage to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include : 
+
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: + translations system ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+* broadcasted and thematics channels,
+* live streaming and video catchup encoding and playout,
+* video resuming,
+* local adservers (for videos and display), DMP system and CDN,
+* contribution via our backoffice and automated thought our API,
+* and almost all the features of the 6play application.
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: s/application/platform

File: _posts/2018-03-26-6play-goes-international.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+  credit: m6web
+comments: true
+language: en
+---
+
+
+Within less than a year m6web and [techm6web](https://twitter.com/TechM6Web) managed to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe, all based on the [6play](https://www.6play.fr) technology. 
+
+Theses deployments include: 
+
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: + translations system ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+* broadcasted and themathic channels,
+* live streaming and video catchup encoding and playout,
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: s/live streaming and video catchup encoding and playout/live streaming, video catchup encoding and playout

File: _posts/2018-03-26-6play-goes-international.md [hunk 15]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "the 6play platform goes international"
+description: ""
+author:
+  name:  Olivier Mansour
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: TechM6Web du coup ici vu le coté "global de l'annonce" ?

File: _posts/2018-03-26-6play-goes-international.md [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+category: 6play
+tags: [6play, intl]
+image:
+  feature: 6play-intl/feature.png
+  credit: m6web
+comments: true
+language: en
+---
+
+
```
Comment (kennydee) [inline _posts/2018-03-26-6play-goes-international.md]: Un retour de ligne en trop

File: _posts/2018-03-26-6play-goes-international.md [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+* live streaming, video catchup encoding and playout,
+* video resuming,
+* local adservers (for videos and display), DMP system and CDN,
+* contribution via our backoffice and automated through our API,
+* translations system,
+* and almost all the features of the 6play platform.
+
+## RTL Play (Belgium)
+
+![RTL Play Belgium](/images/posts/6play-intl/rtl_play_be.png)
```
Comment (pabedu) [inline _posts/2018-03-26-6play-goes-international.md]: I don't see rtl_most used in tour links and paths don't seem correct :)

File: _posts/2018-03-26-6play-goes-international.md [hunk 18]
```diff
@@ -0,0 +1,43 @@
...
+category: 6play
+tags: [6play, intl]
+image:
+  feature: 6play-intl/feature.png
+  credit: m6web
+comments: true
+language: en
+---
+
+Within less than a year m6web and [techm6web](https://twitter.com/TechM6Web) managed to launch three websites, three android apps, three ios apps for three RTL TV channel in Europe, all based on the [6play](https://www.6play.fr) technology.
```
Comment (pabedu) [inline _posts/2018-03-26-6play-goes-international.md]: lots of three ;) 🥉 maybe "managed to launch websites, android apps and ios apps for three RTL TV channels in Europe"

File: _posts/2018-03-26-6play-goes-international.md [hunk 19]
```diff
@@ -0,0 +1,43 @@
...
+
+![RTL Play Belgium](/images/posts/6play-intl/rtl_play_be.png)
+
+## RTL Play (Croatia)
+
+![RTL Play Croatia](/images/posts/6play-intl/rtl_play_hr.png)
+
+## RTL Most (Hungary)
+
+![RTL Play Hungary](/images/posts/6play-intl/rtl_play_hu.png)
```
Comment (pabedu) [inline _posts/2018-03-26-6play-goes-international.md]: /images/6play-intl/rtl_most.png nope ?

**Pull request merged**